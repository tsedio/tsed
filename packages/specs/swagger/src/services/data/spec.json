{
  "info": {
    "description": "This is a sample server Petstore server.  You can find out more about     Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).      For this sample, you can use the api key `special-key` to test the authorization     filters.",
    "title": "Swagger Petstore",
    "termsOfService": "http://swagger.io/terms/",
    "contact": {
      "email": "apiteam@swagger.io"
    },
    "license": {
      "name": "Apache 2.0",
      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
    }
  },
  "securityDefinitions": {
    "global_auth": {
      "type": "oauth2",
      "authorizationUrl": "http://petstore.swagger.io/oauth/dialog",
      "flow": "implicit",
      "scopes": {
        "write:global": "modify pets in your account",
        "read:global": "read your pets"
      }
    },
    "calendar_auth": {
      "type": "oauth2",
      "authorizationUrl": "http://petstore.swagger.io/oauth/dialog",
      "flow": "implicit",
      "scopes": {
        "write:calendar": "modify pets in your account",
        "read:calendar": "read your pets"
      }
    }
  }
}