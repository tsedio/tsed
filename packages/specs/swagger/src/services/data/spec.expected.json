{
  "consumes": [
    "application/json"
  ],
  "info": {
    "contact": {
      "email": "apiteam@swagger.io"
    },
    "description": "This is a sample server Petstore server.  You can find out more about     Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).      For this sample, you can use the api key `special-key` to test the authorization     filters.",
    "license": {
      "name": "Apache 2.0",
      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
    },
    "termsOfService": "http://swagger.io/terms/",
    "title": "Swagger Petstore",
    "version": "1.0.0"
  },
  "produces": [
    "application/json"
  ],
  "securityDefinitions": {
    "calendar_auth": {
      "authorizationUrl": "http://petstore.swagger.io/oauth/dialog",
      "flow": "implicit",
      "scopes": {
        "read:calendar": "read your pets",
        "write:calendar": "modify pets in your account"
      },
      "type": "oauth2"
    },
    "global_auth": {
      "authorizationUrl": "http://petstore.swagger.io/oauth/dialog",
      "flow": "implicit",
      "scopes": {
        "read:global": "read your pets",
        "write:global": "modify pets in your account"
      },
      "type": "oauth2"
    }
  },
  "swagger": "2.0"
}